transport/kubernetes-jobmonitor/src/test/kotlin/ReaperTest.kt

```diff
@@ -63,23 +63,23 @@ class ReaperTest : WordSpec({
         "periodically query completed jobs" {
             val zoneOffset = ZoneOffset.ofHours(2)
             val date1 = OffsetDateTime.of(
-                2023,
-                Month.MAY.value,
-                3,
-                12,
-                8,
-                11,
-                0,
+                /* year = */ 2023,
```

Comment:
Why the named parameters are commented ?

Reply:
Because that's a Java function being called, which does not support named parameters natively.

Reply:
TIL